.github/workflows/ci.yml

```
@@ -83,8 +85,7 @@ jobs:
         apt:
           - libopenjp2-7
       envs: |
-        - linux: base_deps
-        - linux: py39-devdeps
+        - windows: py310
```

```suggestion
        - linux: py310
```

Windows is slower anyway skimage issues aside.

---

I will fix this in my open PR